crates/turborepo-lockfiles/src/berry/ser.rs

```diff
@@ -163,6 +163,9 @@ where
 
     for (dependency, meta) in metadata {
         let dependency = dependency.as_ref();
+        if meta.built.unwrap_or_default() {
```

Comment:
Sorry I missed this, I believe this logic needs to be changed so we only include `built` if it is present. `add_line` will also need to be updated to be able to write both true and false.
```suggestion
        if meta.built.unwrap_or_default() {
```

Reply:
I updated it so all three dependencyMeta settings support true and false. And check if they are present before adding the lines.

Let me know if you find any more 🐛 s. 🙇🏼‍♂️